forms-flow-web/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# base image
+FROM node:12.4.0-alpine as build-stage
```

Comment:
If there is no problem with dependecies I suggest using the version of node LTS that is currently in ACTIVE state which is v16. 

v12 maintenance support will finish in April and v14 support for security update will finish in 1 year and 3 months.

https://endoflife.date/nodejs

Reply:
Unfortunately moving to node 16 introduces some dependency issues, so think we'll have to stick with `v14`